src/app/profile/overview/work-items/work-items.component.ts

```diff
@@ -85,32 +95,35 @@ export class WorkItemsComponent implements OnDestroy, OnInit  {
   }
 
   fetchWorkItems(): void {
-    this.currentSpace = this.getSpaceById(this.currentSpaceId);
-    if (this.currentSpace != null) {
+    this.currentSpace = this.spaces.find((s: Space) => s.id == this.currentSpaceId);
+    if (this.currentSpace !== undefined) {
       this.fetchWorkItemsBySpace(this.currentSpace);
     }
   }
 
   private fetchWorkItemsBySpace(space: Space): void {
-    let filters = [{
-      paramKey: 'filter[assignee]',
-      value: this.context.user.id,
-      active: true
-    }];
+    let filters = [
+      {
+        paramKey: 'filter[assignee]',
+        value: this.context.user.id,
+        active: true
+      }
+    ];
     this.workItemService._currentSpace = space;
-    this.subscriptions.push(this.workItemService
-      .getWorkItems(100000, filters).subscribe(result => {
-        this.workItems = filterOutClosedItems(result.workItems);
-      })
+    this.subscriptions.push(
+      this.workItemService
+        .getWorkItems(100000, filters)
+        .subscribe(
+          (result: {
+            workItems: WorkItem[];
+            nextLink: string;
+            totalCount?: number;
+            included?: WorkItem[];
+            ancestorIDs?: string[];
+          }) => {
```

Comment:
Ouch... I expected the Planner module to export a type definition for this. Maybe that's something we should request they add so we can clean this up later.